.github/workflows/mvn-verify-check/populate-daily-cache.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Copyright (c) 2024, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -x
+max_retry=3; delay=30; i=1
+while true; do
+for pom in pom.xml 
+do
+    mvn $COMMON_MVN_FLAGS --file $pom help:evaluate -pl dist \
+    -Dexpression=included_buildvers \
+    -DforceStdout -PnoSnapshots -q | tr -d ',' | \
```

Comment:
`-PnoSnapshots` is for scala213, be sure about the shims list you are referring for different scala versions